src/main/java/spring/practice/config/WebConfig.java

```diff
@@ -0,0 +1,22 @@
+package spring.practice.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import org.springframework.web.servlet.view.JstlView;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages = {"spring.practice.controllers"})
+public class WebConfig {
+    @Bean
+    public InternalResourceViewResolver resolver() {
```

Comment:
```suggestion
    public InternalResourceViewResolver getResolver() {
```
назва методу - дієслово або фраза на його основі(get, put, createNewFile)

Reply:
ok